src/Resources/views/storefront/page/checkout/confirm/confirm-payment.html.twig

```diff
@@ -46,6 +48,12 @@
              data-adyen-plugin-id="{{ adyenFrontendData.pluginId }}"
              data-selected-payment-method-handler="{{ adyenFrontendData.selectedPaymentMethodHandler }}"
              data-selected-payment-method-plugin-id="{{ adyenFrontendData.selectedPaymentMethodPluginId }}"
+             data-name="{{ adyenFrontendData.name }}"
+             data-address-line="{{ adyenFrontendData.addressLine }}"
+             data-city="{{ adyenFrontendData.city }}"
+             data-post-code="{{ adyenFrontendData.postCode }}"
+             data-country-code="{{ adyenFrontendData.countryCode }}"
+             data-phone-number="{{ adyenFrontendData.phoneNumber }}"
```

Comment:
These fields should be removed now right?

Reply:
Yes, my bad. Removed in https://github.com/Adyen/adyen-shopware6/pull/192/commits/b6b6e6086af01dd9f4b1b24c88fb1e39c08ae3f6